sdk/dotnet/Alb/Inputs/ListenerDefaultActionAuthenticateOidcGetArgs.cs

```diff
@@ -13,14 +13,14 @@ namespace Pulumi.Aws.Alb.Inputs
     public sealed class ListenerDefaultActionAuthenticateOidcGetArgs : Pulumi.ResourceArgs
     {
         [Input("authenticationRequestExtraParams")]
-        private InputMap<object>? _authenticationRequestExtraParams;
```

Comment:
This is part of the same code import - it means that we need to pass strings in the maps